config/routes.rb

```diff
@@ -1,60 +1,13 @@
 Rails.application.routes.draw do
-  root 'homes#index'
   devise_for :users
 
-  resources :tasks
-  # The priority is based upon order of creation: first created -> highest priority.
-  # See how all your routes lay out with "rake routes".
-
-  # You can have the root of your site routed with "root"
-  # root 'welcome#index'
-
-  # Example of regular route:
-  #   get 'products/:id' => 'catalog#view'
-
-  # Example of named route that can be invoked with purchase_url(id: product.id)
-  #   get 'products/:id/purchase' => 'catalog#purchase', as: :purchase
-
-  # Example resource route (maps HTTP verbs to controller actions automatically):
-  #   resources :products
+  resources :projects do
+    resources :tasks
+  end
 
-  # Example resource route with options:
-  #   resources :products do
-  #     member do
-  #       get 'short'
-  #       post 'toggle'
-  #     end
-  #
-  #     collection do
-  #       get 'sold'
-  #     end
-  #   end
+  authenticated :user do
+    root 'projects#index', as: 'authenticated_root'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.